ldap-connector-test/src_test/resources/docker/ldifs/data.ldif

```diff
@@ -0,0 +1,36 @@
+version: 1
```

Comment:
these data.ldif and octopus.ldif files are now twice in the SCM ... we should only maintain one version.

Reply:
The folder "ldifs" is used to set up default db for testing (like person_query or group_query test case) while octopus.ldif is additional db schema of this instance (like group and user class). 

I tried to combine 2 of them 1 one file but there is no hint to that.

How if I rename the schema file to make it more meaningful & not make any confuse for us?

Reply:
it's totally ok to have these 2 files. 
But it seems as if you have the same files twice under version control:
- src_test/resources/docker/ldifs 
- docker/ldifs
IMHO one of these directories should be removed.

Reply:
Oops, my bad. I will check & remove one of these folders soon.